### ISSUE TITLE
Issue sorting columns

### ISSUE THREAD

**Issue by starettie** [JohnTendik/jtrt-tables#35]
I imported excel data as CSV file and it worked beautifully. But the columns don't sort. See http://www.baast.ca/oils-chart/  
- the option to sort columns by column headings is selected

**Comment by JohnTendik** [comment]
Hi starettie, 

https://github.com/JohnTendik/jtrt-tables/issues/34

Can you please try this fix and let me know if that fixes your issue? I have included this fix in the next version's code but haven't gone around to releasing it yet. 

Thanks
John

**Comment by starettie** [comment]
Hi John, 

I haven't been able to try this fix because my tables
disappeared from the admin area. They still appear on the front end (see
example http://www.baast.ca/oils-chart/ ), and they show up in the list
of tables in the admin, but when I click view or edit on a table, the
table doesn't load. Any ideas? (The problem occurred after I finished
adding info to a table and clicked 'update') 

On 2017-03-16 13:08, John
Tendik wrote: 

> Hi starettie, 
> 
> #34 [1] 
> 
> Can you please try
this fix and let me know if that fixes your issue? I have included this
fix in the next version's code but haven't gone around to releasing it
yet. 
> 
> Thanks
> John 
> 
> --
> You are receiving this because you
authored the thread.
> Reply to this email directly, view it on GitHub
[2], or mute the thread [3].
 

Links:
------
[1]
https://github.com/JohnTendik/jtrt-tables/issues/34
[2]
https://github.com/JohnTendik/jtrt-tables/issues/35#issuecomment-287125974
[3]
